masschallenge/opsworks-web-python
# AC-7555 Add notification/alerts for testim test suite failures

### Changes introduced in: [AC-7555](https://masschallenge.atlassian.net/browse/AC-7555):
- installs jq on instances

### --> [Sibling PR (accelerate)](https://github.com/masschallenge/accelerate/pull/2609)

### How to test
- Update stack settings to use this branch (AC-7555) for Chef (opsworks-web-python.git)
- On your stack's EC2 instance, run command `update custom cookbooks`, then run command `setup`
- Make a deployment to the stack with release `vTR-AC-7555v58` (Ensure that the necessary variables are given to the stack, the values are recorded in test4)
- A notification is sent to the #testim channel after deployment

#### merge first

expected variables
```
TESTIM_APP_DIRECTORY
TESTIM_TOKEN
TESTIM_PROJECT
TESTIM_TESTING_GRID
TESTIM_BUILD_CHANNEL
TESTIM_BUILD_CHANNEL_WEBHOOK
TESTIM_ECS_CLUSTER
TESTIM_ECS_FAMILY
TESTIM_IMAGE
TESTIM_SUBNET1
TESTIM_SUBNET2
TESTIM_SECURITY_GROUP
TESTIM_ASSIGNPUBLICIP
TESTIM_ECSTASKEXECUTIONROLE
TESTIM_LOGDRIVER
TESTIM_AWSLOGS_GROUP
TESTIM_AWSLOGS_REGION
TESTIM_AWSLOGS_STREAM_PREFIX
```
